app/controllers/provider/admin/dashboards_controller.rb

```diff
@@ -14,6 +14,16 @@ def show
     @services           = current_user.accessible_services
     @messages_presenter = current_presenter
     @unread_messages_presenter = unread_messages_presenter
+
+    @widget_products = @services.order(updated_at: :desc)
+                                .decorate
+                                .take(5)
```

Comment:
if we use `paginate` instead as with the other controllers, how many would there be by default? is it worth it? in that case, it would be easier to DRY this up.

Reply:
https://github.com/3scale/porta/pull/2194/files#r501113662

Reply:
There won't be pagination in the dashboard, only the a list of 5 items.